src/components/charts/SummaryPieChart/index.jsx

```diff
@@ -32,16 +32,33 @@ class SummaryPieChart extends React.Component {
   }
 
   render() {
+    const tempDataArr = [...this.props.data];
+    let tempDataTotal = this.props.totalCount;
+    // if chart is missing total add it
+    const dataSum = tempDataArr.reduce((a, entry) => a + entry.value, 0);
+    if (!tempDataTotal) {
+      tempDataTotal = dataSum;
+
+    // if chart is missing data add it
+    } else if (tempDataTotal > dataSum) {
+      tempDataArr.push({
+        name: 'No Data',
+        value: tempDataTotal - dataSum,
```

Comment:
possible info leak in here: if env is in `regular` tier access level, this value may be less than the `tiered_access_limit` value, and should cause the chart get locked